act-rules-format.bs

```diff
@@ -17,43 +17,41 @@ Introduction {#intro}
+In accessibility, there are often different technical solutions to make the same type of content accessible. For example, there are multiple methods for giving an `img` element in HTML an accessible name. Multiple solutions could be tested in a single rule; however, such a rule tends to be quite complex, making it difficult to understand and maintain. The ACT Rules Format solves this by providing two types of rules:
 
-- <dfn>Atomic rules</dfn> describe how to test a specific type of solution. It contains a precise definition of what elements, nodes or other parts of a [=test subject=] are to be tested, and when those elements are considered to fail the rule. These rules are to be kept small and *atomic*. Meaning that atomic rules test a single "failure condition", and do so without using the [=outcome=] of other rules.
+- <dfn>Atomic rules</dfn> describe how to test a specific type of solution. It contains a precise definition of what elements, nodes or other "parts" of a [=test subject=] are to be tested, and when those elements are considered to fail the rule. These rules are to be kept small and *atomic*. Meaning that atomic rules test a single "failure condition", and do so without using the [=findings=] from other rules.
 
-- <dfn>Composite rules</dfn> describe how the [=outcomes=] of multiple [=atomic rules=] or composite rules can be combined into a single outcome. A composite rule can have multiple "satisfying conditions", each of these provided in separate atomic or composite rules. The logic in the composite rule describes how any one of these satisfying conditions, or some combination of them is used to determine a single outcome.
+- <dfn>Composite rules</dfn> describe how the [=findings=] of multiple [=atomic rules=] or composite rules can be combined into a single [=outcome=]. A composite rule can have multiple "satisfying conditions", each of these tested in separate rules. The logic in the composite rule describes how any one of these satisfying conditions, or some combination of them is used to determine a single outcome.
 
 <div class="example">
-  <p>Composite rule: Header cells in HTML tables (<a href="https://www.w3.org/WAI/WCAG21/quickref/#info-and-relationships" target="_blank">WCAG 2 Success Criterion 1.3.1</a>).</p>
-  <p>This rule uses outcomes from the following rules:</p>
+  <p>Composite rule: video elements have an audio description or media alternative (<a href="https://www.w3.org/WAI/WCAG21/quickref/#audio-description-or-media-alternative-prerecorded" target="_blank">WCAG 2.1, success criterion 1.2.3 Audio Description or Media Alternative</a>).</p>
+  <p>Each HTML `video` element meets expectations from at least one of the following rules:</p>
```

Comment:
Suggested edit:

> Each HTML `video` element meets the expectations defined by at least one of the following rules:

Rationale: "meets expectations" is vague - it could mean meeting only parts of the defined expectation.

Reply:
"meets all expectations"

Reply:
Going with "meets the expectations" as suggested. "All expectations from one rule" is a bit confusing IMO. I'm not using "as defined in", as I don't think it helps readability. This is an example, making this editorial. We can revisit it if we have to in CR.